src/components/sbb-button/sbb-button.stories.jsx

```diff
@@ -290,6 +290,15 @@ IconOnly.args = {
   text: undefined,
 };
 
+export const FormField = Template.bind({});
```

Comment:
I'd prefer to not have any example here because it's not a public api for the button. only used in form field. Alternatively you could create a form field here and also make a test for disabled state and active (data-active) state.